src/main/java/mil/dds/anet/resources/OrganizationResource.java

```diff
@@ -223,4 +227,10 @@ private WebApplicationException handleSqlException(UnableToExecuteStatementExcep
 		}
 		return new WebApplicationException(Status.INTERNAL_SERVER_ERROR);
 	}
+
+	private void validateApprovalSteps(ApprovalStep step) {
```

Comment:
Since this method validates only one step, use a singular method name: `validateApprovalStep`.